### ISSUE TITLE
Link to the Flathub page in the repo is invalid

### ISSUE THREAD

**Issue by CorruptComputer** [flathub/com.discordapp.DiscordCanary#33]
This link on the GitHub repo:
![image](https://user-images.githubusercontent.com/5573038/179368322-c0d27210-fbdc-458f-aeee-3cea31b7edcf.png)

Leads to an "App not found" page on Flathub:
![image](https://user-images.githubusercontent.com/5573038/179368341-47a49519-7b05-44c3-ba7e-a5bdc01d8c88.png)

**Comment by ghost** [comment]
Also can't seem to download it via Flatpak's command line tool...

**Comment by CorruptComputer** [comment]
@allinkdev That is probably because you do not have the Flathub beta repo enabled.

See: https://discourse.flathub.org/t/how-to-use-flathub-beta/2111

**Comment by NicTanghe** [comment]
Link is does not work.

**Comment by refi64** [comment]
I don't think this is really in our control, the URLs are set automatically but beta channel apps don't appear in the website.
